shavar/lists.py

```diff
@@ -45,9 +45,15 @@ def add_versioned_lists_to_registry(
         branch_name = branch.get('name')
         ver = version.parse(branch_name)
         if isinstance(ver, version.Version):
+            if 'entity/' in settings['source']:
+                original_path = 'entity/'
+                versioned_path = 'entity/{}/'
+            else:
```

Comment:
`else` is a code-sniff to me these days; prefer helper functions and early returns instead.

Can we put this logic into a helper function?
```python
def get_original_and_versioned_paths(source):
    if 'entity/' in source:
        return 'entity/', 'entity/{}/'
    return 'tracking/', 'tracking/{}'
```

```python
original_path, versioned_path = get_original_and_versioned_paths(settings['source'])
```
?